ANNOUNCE

```diff
@@ -37,7 +37,7 @@ The cclib development team
 ———
 
 Summary of changes since last version:
-    New attribute polarizabilities for static or dynamic dipole polarizability
-    New attribute pressure for thermochemistry
-    Detecting closed shells in parsed data
-    Support for Python 3.6
+    * Support for writing Molden and WFX files
+    * Support for thermochesmitry attributes in ORCA
+    * Support for chelpg atomic charges in ORCA (Richard Gowers)
```

Comment:
Do you want to make this more consistent with the other points? Suggest either removing (Richard Gowers) or adding the other contributors.

Reply:
Removed.